Dockerfile

```
@@ -1,7 +1,9 @@
 FROM fedora:36
 
 RUN dnf -y update \
```

```suggestion
RUN dnf -y upgrade \
```
You wanna use `dnf upgrade` because `udpate` is just a legacy alias which is no longer present in `dnf5`.

---

Good catch, thanks! Fixed